src/Sylius/Behat/Context/Setup/SecurityContext.php

```diff
@@ -70,6 +70,7 @@ public function iAmLoggedInAs($email)
 
     /**
      * @Given /^I am logged in customer$/
+     * @Given /^I am logged in as customer$/
```

Comment:
I am a logged in customer
I am logged in as a customer

These 2 seem to be correct. /cc @TheMadeleine?